tests/reproducible_builds/check_reproducible_build.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
+
+if [ -d "/home/$USER/zephyr_src1" ]; then
+    rm -rf /home/$USER/zephyr_src1
+fi
+
+if [ -d "/home/$USER/zephyr_src2" ]; then
```

Comment:
/home/$USER = $HOME
and I would not put anything in home, please always use temporary directories with random names in /tmp or something like that.